### PR TITLE
Video fixes

### PR DESCRIPTION
**Pull request by kamil4**
While working on https://github.com/LycheeOrg/Lychee/issues/675 I noticed that support for various video types was incomplete, especially in the Exiftool reader.

### CODE REVIEW

File: lib/PHPExif/Mapper/Exiftool.php [hunk 1]
```diff
@@ -214,7 +237,7 @@ public function mapRawData(array $data)
                     }
 
                     break;
-                case self::DATETIMEORIGINAL_QUICKTIME:
+                case self::DATETIMEORIGINAL_APPLE:
```
Comment (tmp-hallenser) [inline lib/PHPExif/Mapper/Exiftool.php:240]: I think, there is a testcase for "DATETIMEORIGINAL_QUICKTIME" which needs to be adjusted

File: lib/PHPExif/Adapter/FFprobe.php [hunk 2]
```diff
@@ -94,6 +94,10 @@ public function getExifFromFile($file)
     {
         $mimeType = mime_content_type($file);
 
+        if ($mimeType === 'image/x-tga') {
+            $mimeType = 'video/mpeg';
```
Comment (tmp-hallenser) [inline lib/PHPExif/Adapter/FFprobe.php:98]: That's a funny one :) we should ignore these line for the code coverage calc

**Comment by tmp-hallenser** [comment]
Does ffmpeg handle the other video files correctly?

File: lib/PHPExif/Adapter/FFprobe.php (hunk 2 above)
Comment (kamil4) [reply lib/PHPExif/Adapter/FFprobe.php:98]: It was a pain to figure out! Actually not so much on the php-exif side as in Lychee itself. I mean, who would've thought?! And yeah, I don't think we can simulate this one for coverage...

**Comment by kamil4** [comment]
Yeah, the ffmpeg reader was very robust in my testing!

File: lib/PHPExif/Mapper/Exiftool.php (hunk 1 above)
Comment (kamil4) [reply lib/PHPExif/Mapper/Exiftool.php:240]: Good catch; I just pushed a commit to take care of it!

**Pull request merged**